modules/react/tokens/index.ts

```diff
@@ -64,7 +64,7 @@ export {
   monoFontFamily,
   BrandingColor,
   CanvasDepth,
-  CanvasDepthValue,
+  CanvasDepthValues,
```

Comment:
This name update is more consistent with our other type names: `CanvasBorderRadiusValues`, `CanvasSpaceValues`, `CanvasSpaceNumberValues`, but it will require a codemod to make that update easier for consumers.